static/js/components/JobDetails.jsx

```diff
@@ -190,8 +227,9 @@ var JobDetails = React.createClass({
   },
 
   cronUsage: function() {
+    var msg = "Note: To reduce errors, direct modifications aren't currently supported. Please instead deactivate the old job and add a new.";
```

Comment:
My only thought here is that this repo is open source. Not sure having something Clever specific like "cron jobs at Clever are refreshed every 30 minutes" would be the right thing here. What are your thoughts?